### PR TITLE
refactor(data-toolbar): added content wrapper, updated margins

### LINKED ISSUE #2456: Data-toolbar CSS update
Screen Shot 2019-11-21 at 10 33 37 AM

Need to update CSS to remove `margin-right` on all `.pf-c-data-toolbar__expandable-content .pf-c-data-toolbar__group`s not `.pf-m-chip-container`

### PR DESCRIPTION
**Pull request by mattnolting**
Closes #2456 

- Toolbar needs a wrapper to separate static content from dynamically generated content
- Margins for groups and items need refinement

### CODE REVIEW

**Comment by mcoker** [comment]
Looks like we need to update the demos with these changes, too.

File: src/patternfly/components/DataToolbar/data-toolbar.scss [hunk 1]
```diff
@@ -397,3 +408,8 @@ $pf-c-data-toolbar--spacer-map: build-spacer-map("none", "sm", "md", "lg");
     }
   }
 }
+
+// Override .pf-m-toggl-group margin modifier
```
Comment (mcoker) [inline src/patternfly/components/DataToolbar/data-toolbar.scss]:
```suggestion
// Override .pf-m-toggle-group margin modifier
```

File: src/patternfly/components/DataToolbar/examples/DataToolbar.md [hunk 2]
```diff
@@ -66,18 +56,30 @@ Several components in the following examples do not include functional and/or ac
...
-      Item
-    {{/data-toolbar-item}}
-    {{#> data-toolbar-item}}
-      Item
-    {{/data-toolbar-item}}
-    {{#> data-toolbar-item}}
-      Item
-    {{/data-toolbar-item}}
-  {{/data-toolbar-content}}
+      {{#> data-toolbar-item data-toolbar-item--modifier="pf-m-separator"}}{{/data-toolbar-item}}
```
Comment (mcoker) [inline src/patternfly/components/DataToolbar/examples/DataToolbar.md]: de-dent this line

File: src/patternfly/components/DataToolbar/examples/DataToolbar.md [hunk 3]
```diff
@@ -290,43 +302,45 @@ The `.pf-m-toggle-group` controls when, and at which breakpoint, filters will be
 | Class | Applied to | Outcome |
 | -- | -- | -- |
 | `.pf-m-show{-on-[breakpoint]}` | `.pf-c-data-toolbar__group.pf-m-toggle-group`, `.pf-c-data-toolbar__expandable-content` | Modifies toolbar element visibility at breakpoint. This selector must be applied consistently to toggle group and expandable content. |
-| `.pf-m-toggle-group-container` | `.pf-c-data-toolbar__content` | Modifies the data toolbar content for toggle group, expandable content layout. |
 | `.pf-m-chip-container` | `.pf-c-data-toolbar__content`, `.pf-c-data-toolbar__group` | Modifies the data toolbar element for applied filters layout. |
+| `.pf-m-chip-container` | `.pf-c-data-toolbar__content`-section, `.pf-c-data-toolbar__group` | Modifies the data toolbar element for applied filters layout. |
```
Comment (mcoker) [inline src/patternfly/components/DataToolbar/examples/DataToolbar.md]: this is already on the line above it - do you just need to add `-section` to the `.pf-c-data-toolbar__content` selector?

**Comment by mattnolting** [comment]
@mcoker @christiemolloy updated

**Pull request merged**